build.gradle

```diff
@@ -8,12 +8,12 @@
 
 buildscript {
     ext.kotlin_version = '1.3.31'
-    ext.android_components_version = '0.50.0'
+    ext.android_components_version = '0.52.0'
```

Comment:
I thought it made sense to track the latest tag, since it's a ways off until we integrate anyway.